google/cloud/baremetalsolution/quickstart/WORKSPACE.bazel

```diff
@@ -23,9 +23,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # NOTE: Update this version and SHA256 as needed.
 http_archive(
     name = "com_github_googleapis_google_cloud_cpp",
-    sha256 = "a7e51bfffb95a377094b2ae7e3b9f715a68ed931c48992c7273b2fae989c029c",
-    strip_prefix = "google-cloud-cpp-1.36.0",
-    url = "https://github.com/googleapis/google-cloud-cpp/archive/v1.36.0.tar.gz",
```

Comment:
hm...

Reply:
Yeah... unless we teach renovate bot to update the generate_scaffold.cc file too, this will keep happening

Reply:
good idea. I'll send a PR